models/stg_linkedin.yml

```diff
@@ -61,7 +61,7 @@ models:
     tests:
       - dbt_utils.unique_combination_of_columns:
           combination_of_columns:
-            - version_tag
+            - _fivetran_synced
```

Comment:
This _should_ be the `last_modified_at`/`last_modified_timestamp` column, however, there are currently duplicate records.

Reply:
Coming back to this comment, if I recall you mentioned there was a rollout on the connector side that should have deleted these duplicate records. These tests _should_ catch and raise duplicates, so aren't we intentionally skipping those with the addition of the _fivetran_synced field?

My proposal would be to either remove _fivetran_synced and catch the true duplicates so customers may be aware and address them, or remove the test altogether. What are your thoughts?

Reply:
So I took some time to look into this a bit more today - in our data, it looks like there could still be duplicates if the creative object was deleted from the source and therefore no longer synced into the destination.

I don't think engineering plans on removing duplicates in the above instances and therefore even if the test captured duplicates, I'm not sure if there would be an action item. 🤔 While it doesn't 100% feel right for me to just remove the test all together, I don't know what other options we truly have here.

Reply:
Removed and added messaging within the CHANGELOG!